back/app/interactors/application_interactor.rb

```diff
@@ -0,0 +1,30 @@
+class ApplicationInteractor
+  include Interactor
```

Comment:
Even after reading the documentation of the gem, I fail to see what interactors are useful for 😅 Perhaps it will become clear later during the review.

Reply:
I read through the documentation very briefly though :p

Reply:
I really like it, it has some very useful features like queuing interactions or `rollback` when a later interaction fails, e.g.:


```ruby
class SomeOrganizer < ApplicationOrganizer
  organize Interactor1, # <= calls the 2 interactors in succession, but you can still call them individually
                  Interactor2
end
```

```ruby
class Interactor1 < ApplicationInteractor
  def call
    # does some operation
  end
  
  def rollback
    # method that gets called if Interactor 1 or 2 fail.
  end
end
```

Reply:
Thanks for explaining :) Seems like a useful pattern indeed :)